crates/unleash-edge-http-client/src/lib.rs

```diff
@@ -103,34 +104,15 @@ lazy_static! {
     .unwrap();
 }
 
+#[cfg_attr(test, derive(Default))]
```

Comment:
Also not safe to default, so I've moved the default to a compiler gate behind test. Gives us test code that can mess with this but not prod code